SPARQLWrapper/Wrapper.py

```diff
@@ -1254,32 +1268,32 @@ def _content_type_in_list(real, expected):
                     RuntimeWarning,
                 )
                 return ct
-        return None
 
-    def print_results(self, minWidth=None):
+    def print_results(self, minWidth: Optional[int] = None) -> None:
         """This method prints a representation of a :class:`QueryResult` object that MUST has as response format
         :data:`JSON`.
 
         :param minWidth: The minimum width, counting as characters. The default value is ``None``.
-        :type minWidth: string
+        :type minWidth: int
         """
 
         # Check if the requested format was JSON. If not, exit.
         responseFormat = self._get_responseFormat()
         if responseFormat != JSON:
             message = "Format return was %s, but JSON was expected. No printing."
             warnings.warn(message % (responseFormat), RuntimeWarning)
-            return
+            return None
 
         results = self._convertJSON()
-        if minWidth:
+        if isinstance(minWidth, int):
```

Comment:
This looks like it can go into a separate PR.

Reply:
934e334